### PR TITLE
add: #38 start specific track at specific position

### PR DESCRIPTION
**Pull request by digitaldrummerj**
fixes: #38

### CODE REVIEW

File: src/helpers.ts [hunk 1]
```diff
@@ -244,20 +244,22 @@ export async function PlaySpecificTracks(
 	instance: SpotifyInstanceBase,
 	deviceId: string,
 	uris: string[],
-	attempt = 0
+	attempt = 0,
+	positionMs = 0
```
Comment (Julusian) [inline src/helpers.ts:248]: This parameter should be before attempt, probably without a default value.

File: src/actions.ts [hunk 2]
```diff
@@ -112,13 +112,22 @@ export function GetActionsList(executeAction: (fcn: DoAction) => void): Companio
...
+					max: 1000000,
+				},
 			],
 			callback: (action) => {
 				if (typeof action.options.tracks === 'string') {
 					const tracks = action.options.tracks.split(',').map((track) => 'spotify:track:' + track.trim())
 
 					executeAction(async (instance, deviceId) => {
-						if (deviceId) await PlaySpecificTracks(instance, deviceId, tracks)
+						if (deviceId) await PlaySpecificTracks(instance, deviceId, tracks, 0, Number(action.options.positionMs))
```
Comment (Julusian) [inline src/actions.ts:130]: I think this should be `Number(action.options.positionMs) || 0` to make sure that old actions where the option is undefined.

**Review by Julusian — COMMENTED** [review]
Thanks for doing this, just a couple of tiny comments :)

File: src/helpers.ts (hunk 1 above)
Comment (digitaldrummerj) [reply src/helpers.ts]: @Julusian updated

File: src/actions.ts (hunk 2 above)
Comment (digitaldrummerj) [reply src/actions.ts]: @Julusian updated

**Pull request merged**